catalogs/catalog-lakehouse-paimon/build.gradle.kts

```diff
@@ -122,12 +122,14 @@ dependencies {
   testImplementation(libs.junit.jupiter.api)
   testImplementation(libs.mysql.driver)
   testImplementation(libs.postgresql.driver)
+  testImplementation(libs.h2db)
   testImplementation(libs.bundles.log4j)
   testImplementation(libs.junit.jupiter.params)
   testImplementation(libs.paimon.s3)
   testImplementation(libs.paimon.spark)
   testImplementation(libs.testcontainers)
   testImplementation(libs.testcontainers.localstack)
+  testImplementation(libs.testcontainers.mysql)
```

Comment:
why adding this?

Reply:
> why adding this?

to test paimon catalog when the catalog backend is mysql.

Reply:
ok